### DISCUSSION TITLE
What is the equivalent validation of CEF in phase4

### DISCUSSION THREAD

**Discussion by duke-nuke** [phax/phase4#280] [Q&A] [answered]
Hi Phax 

what is the equivalent validation of the following validator from CEF site 
https://www.itb.ec.europa.eu/invoice/upload
![image](https://github.com/user-attachments/assets/abe0abb3-aae9-4e3a-932e-cfdd314676e8)

when i am validation same invoice with local PeppolValidation2024_05.VID_OPENPEPPOL_INVOICE_UBL_V3 
and their site validator i am getting different results .

which validator config i can use on phase4 to get consistent resuts . 

Regards

**Comment by phax** [comment, marked as answer]
Use one of:
* `eu.cen.en16931:ubl:1.3.12` - EN 16931 UBL Invoice 1.3.12
* `eu.cen.en16931:ubl-creditnote:1.3.12` - EN 16931 UBL Credit Note 1.3.12

hth

**Reply by phax** [reply]
For version 1.3.13 use of course
* `eu.cen.en16931:ubl:1.3.13` - EN 16931 UBL Invoice 1.3.13
* `eu.cen.en16931:ubl-creditnote:1.3.13 - EN 16931 UBL Credit Note 1.3.13

Requires phive-rules 3.2.1 or later

**Reply by duke-nuke** [reply]
Thanks

We have to add dependency to project phive-rules-en16931 and use class EN16931Validation

Regards
